src/server.js

```diff
@@ -1,13 +1,46 @@
-const express = require("express")
-const morgan = require("morgan")
-const cors = require("cors")
-const app = express()
+const express = require("express");
+const morgan = require("morgan");
+const cors = require("cors");
+const app = express();
+let contacts = require("../data/contacts.js");
+let meetings = require("../data/meetings.js");
 
-app.use(morgan("dev"))
-app.use(cors())
-app.use(express.json())
+app.use(morgan("dev"));
+app.use(cors());
+app.use(express.json());
 
 // write your app code here
 
+app.get("/contacts", function (req, res) {
+  res.status(200).json({ contacts: contacts });
+});
 
-module.exports = app
+app.post("/contacts", function (req, res) {
+  const postContact = req.body;
```

Comment:
it's better practice to extract out the variables, to avoid accepting any input that may come from the POST body:

```js
const { title, content } = req.body
```

If for example `req.body = {title, content, someUnknownProperty}` with the code in my example, the `someUnknownProperty` is ignored, but with your code it would be included.